aws/extensions/computetask_awswin_cfninit/extension.ftl

```diff
@@ -0,0 +1,64 @@
+[#ftl]
+
+[@addExtension
+    id="computetask_awswin_cfninit"
+    aliases=[
+        "_computetask_awswin_cfninit"
+    ]
+    description=[
+        "Runs the cfn init process based on the provided cfninit metadata"
+    ]
+    supportedTypes=[
+        EC2_COMPONENT_TYPE,
+        ECS_COMPONENT_TYPE,
+        COMPUTECLUSTER_COMPONENT_TYPE,
+        BASTION_COMPONENT_TYPE
+    ]
+    scopes=[
+        COMPUTETASK_EXTENSION_SCOPE
+    ]
```

Comment:
This now needs to include handling of a second cfn-init pass for when we need to handle cloudformation actions that depend on the ec2 instance

Reply:
Actually looks like you've added another function for that. This one can be removed. We either have wait or autoscale group now